### ISSUE TITLE
[Bug]: database switcher missing on aws elasticache

### ISSUE THREAD

**Issue by Jimbolino** [RedisInsight/RedisInsight#1730]
**Preconditions**
- add ElastiCache -> Redis Cluster

**Steps to reproduce** 
- connect to the server using ssh tunnel eg: main.xxxxxx.ng.0001.euc1.cache.amazonaws.com:6379

**Actual behavior** 
- no database switcher in topbar

**Expected behavior** 
- there should be a database switcher
(preferably a dropdown, instead of an input field)

**Additional context**
- Engine version 6.2.6
- Number of nodes 3
- Shards 1
- Cluster mode Disabled
- Multi-AZ Disabled

**Comment by ViktarStarastsenka** [comment]
can you please run `CONFIG GET DATABASES` and `INFO KEYSPACE`?

**Comment by Jimbolino** [comment]
```
 > CONFIG GET DATABASES
"ERR unknown command `CONFIG`, with args beginning with: `GET`, `DATABASES`, "

 > INFO KEYSPACE
# Keyspace
db0:keys=311,expires=311,avg_ttl=18022600
db1:keys=734,expires=727,avg_ttl=355058635
db2:keys=6833,expires=5973,avg_ttl=25247895
db3:keys=1,expires=0,avg_ttl=0
db5:keys=2,expires=0,avg_ttl=0
db8:keys=6199,expires=0,avg_ttl=0
```

**Comment by ViktarStarastsenka** [comment]
right, currently we rely on the `CONFIG GET DATABASES`, but it will be improved in late February.

**Comment by ViktarStarastsenka** [comment]
@Jimbolino , it is available with RedisInsight 2.20 (released today). Please re-open the ticket if the issue is still reproducible.
